frontend/src/views/TE/Task.vue

```diff
@@ -190,9 +196,9 @@ export default class Task extends Mixins<TaskMixin, FormatTime>(
   FormatTime
 ) {
   task: any = {};
-  isFetchingTask = true;
+  TaskState = TaskState;
```

Comment:
Can we avoid this duplication?

Reply:
To use the enum in the template I think it needs to be declared in the class.